src/components/Cart/__tests__/Cart.test.js

```diff
@@ -1,11 +1,15 @@
 import React from 'react';
-import renderer from 'react-test-renderer';
+import { create } from 'react-test-renderer';
 
 import Cart from '../Cart';
 
+jest.mock('react-router-dom', () => ({
+  Link: 'Link'
```

Comment:
What for do we need this? I don't see usage in this file?

Reply:
Link is a `react-router-dom` component, that needs to be rendered only inside the `Router` component. Because of our issues with snapshots we haven't manage yet to shallow render in tests, Link needs to be mocked.